.lando.yml

```diff
@@ -60,6 +63,9 @@ tooling:
     service: appserver
   svn:
     service: appserver
+  svn-git-up:
+    service: appserver
+    cmd: "bash $LANDO_MOUNT/bin/svn-git-up"
```

Comment:
I think it would be neat if we could instead make this available as regular a `git` alias command that isn't tied to Lando. With this PR, `git` is our primary tool anyway. On installation we could provision a `.gitconfig` file that contains an `[alias]` section with that command.

Maybe rename the file to `git-svn-checkout` and then have the alias work as `git svn-checkout [<branch>]` (argument similar as now)?

Reply:
What about just `git svn-up [<branch>]`?

Reply:
See 577839c.

I like `up` instead of `checkout` because it the normal operation is going to be to “update” the currently checked-out branch. When supplying an argument, then it's like “update to” another branch. On the other hand, in SVN `checkout` only is used when first “cloning” the repo, whereas in Git it is used for switching branches (which for SVN `switch` is used).

I also like `svn-git-up` because it is shorter, _and_ it's like shouting the command: “SVN, GET UP!!!”

Reply:
My thoughts were that we're using the `git` command primarily, and for that, the command does what is known as `checkout`. The `up` is common for `svn`, therefore I would go with `checkout`. But I'm okay with `git svn-up`.

I think the most proper thing would be to write `git checkout [<branch>] --with-svn` or something like that, but that isn't how `git` aliases work, and it would be way too verbose and long. :)

Reply:
I'd prefer not to have the `svn-git-up` command in Lando, since we already have it via `git svn-up`. _If_ we need to maintain it as `lando` command as well, I think it should be `git-svn-up` to resemble the `git` alias more.